content/manager/service-logs.md

```diff
@@ -104,33 +104,8 @@ At most, 7 rotated files are kept for each rotated log file.
 
 Cloudify logs and events can be removed from the database using a script.
```

Comment:
You can remove Cloudify logs and events from the database with the _delete_logs_and_events_from_db.py_ that is located on the Cloudify Manager under _/etc/cloudify_. When you run the script, the logs and events that exist from after the save period configured in the script are removed.
You can change the _DEFAULT_SAVE_PERIOD_ parameter in the script to set the number of days that are kept in the database (Default: 5). For example, if the _DEFAULT_SAVE_PERIOD_ is 7, logs and events that are older than 7 days are deleted.